JSIL/AST/JSAstVisitor.cs

```diff
@@ -19,6 +19,21 @@ public struct EnclosingNode<T> {
             public string ChildName;
         }
 
+        public class NodeProcessedEventArg : EventArgs
```

Comment:
Normally sticking with convention is good, but an allocation per node visited is pretty nasty perf-wise. GC is a major bottleneck in JSIL :(